shodan/cli/helpers.py

```diff
@@ -10,6 +10,11 @@
 
 from .settings import SHODAN_CONFIG_DIR
 
+try:
+    basestring            # Python 2
+except NameError:
+    basestring = (str, )  # Python 3
```

Comment:
Why a tuple?

```suggestion
    basestring = str  # Python 3
```

Reply:
Because __basestring__ is a tuple of __(str, unicode)__.

Reply:
Oh interesting and I also didn't know that isinstance accepted a tuple! Also ran the changes and it works as expected so thank you.